src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,54 @@
+.sections {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  counter-reset: day-number;
+  flex-wrap: wrap;
+  display: flex;
+  width: $my-var;
+  gap: $width-gap;
```

Comment:
The variable `$width-gap` is also not descriptive enough. Consider renaming it to something more meaningful, like `$calendar-day-gap`.